app/javascript/utils/time.ts

```diff
@@ -2,6 +2,6 @@ import dayjs from 'dayjs'
 import RelativeTime from 'dayjs/plugin/relativeTime'
 dayjs.extend(RelativeTime)
 
-export function fromNow(dateTime) {
+export function fromNow(dateTime: dayjs.ConfigType) {
```

Comment:
Very good. Should be able to import this type too:

```typescript
import type { ConfigType } from 'dayjs'
```

In this case not that much difference, but sometimes you want to do this explicitly, because 

1. it's easier to strip `import type` 
2. doesn't convolute types on namespaces with properties on namespaces/objects/default export bags.

Reply:
Great!